desktop-app/src/renderer/store/features/renderer/index.ts

```diff
@@ -27,7 +32,7 @@ const urlFromQueryParam = () => {
 const initialState: RendererState = {
   address: urlFromQueryParam() ?? window.electron.store.get('homepage'),
   zoomFactor: zoomSteps[window.electron.store.get('renderer.zoomStepIndex')],
-  rotate: false,
+  rotatedDevices: { allRotated: false, devices: {} },
```

Comment:
@WayneRocha Just speaking my thoughts here, I kind of feel like this implementation with `rotatedDevices` is a little more complicated than this functionality needs to be. We're relying too much on the Redux state here while I have a possible solution without that, let me know your thoughts on the below approach. 

Instead of adding `rotatedDevices` state in the redux store to maintain the rotation, we can retain the implementation of the main toolbar rotate button as such with a boolean field in the redux store and for the individual devices we can maintain a local state at the `Device` component, that can decide if the device is individually rotated. 

And then pass that state value and the setter function to the `Device/Toolbar.jsx` component and make it functional. 
And inside the `Device` component, we can use both the redux store boolean value and this local state value and decide if the device needs rotation or now. 

Wdyt? 
I would prefer keeping it simple for easier development on top of this in the future. Please feel free to ping me on Discord if you have any question on this approach.

Reply:
@manojVivek that was a good ideia to simplify the code.
Wdyt about the changes?